README.md

```diff
@@ -11,7 +11,7 @@ Join our community on [`#kapitan`](https://kubernetes.slack.com) or visit [**`ht
 
 How is it different from [`Helm`](https://github.com/kubernetes/helm)? Please look at our [FAQ](#faq)!
 
-<img src="https://raw.githubusercontent.com/deepmind/kapitan/master/docs/kapitan_logo.png" width="250">
+<img src="./docs/kapitan_logo.png" width="250">
```

Comment:
does this work?

Reply:
It worked on my local markdown preview. Also, github seems to parse it fine https://github.com/deepmind/kapitan/blob/2b36971ece1d393b416e6447d5b331e4e266a387/README.md (this is from my PR if i'm not mistaken)